192.168.25.221 - - [08/Feb/2024 13:55:39]  "GET /ProjectMgt/0.0.1/Project/GetProjectByReferenceNumber?ReferenceNumber=UNRA/WRKS/2018-2019/00178 http" 
Traceback (most recent call last):
  File "D:\dev\Apps\Project Management\project-management-api\helpers\utils.py", line 81, in _do_select
    cursor.execute(query, values)
pyodbc.ProgrammingError: ('42000', "[42000] [Microsoft][ODBC Driver 17 for SQL Server][SQL Server]Could not find stored procedure 'getProjectByReferenceNumber'. (2812) (SQLExecDirectW)")
~
192.168.25.221 - - [08/Feb/2024 13:56:39]  "GET /ProjectMgt/0.0.1/Project/GetProjectByReferenceNumber?ReferenceNumber=UNRA/WRKS/2018-2019/00178 http" 
Traceback (most recent call last):
  File "D:\dev\Apps\Project Management\project-management-api\helpers\utils.py", line 81, in _do_select
    cursor.execute(query, values)
pyodbc.ProgrammingError: ('42000', "[42000] [Microsoft][ODBC Driver 17 for SQL Server][SQL Server]Could not find stored procedure 'getProjectByReferenceNumber'. (2812) (SQLExecDirectW)")
~
192.168.25.221 - - [08/Feb/2024 13:58:04]  "GET /ProjectMgt/0.0.1/Project/GetProjectByReferenceNumber?ReferenceNumber=UNRA/WRKS/2018-2019/00178 http" 
Traceback (most recent call last):
  File "D:\dev\Apps\Project Management\project-management-api\helpers\utils.py", line 48, in pyodbc_connect
    database=config.db_name
pyodbc.InterfaceError: ('28000', "[28000] [Microsoft][ODBC Driver 17 for SQL Server][SQL Server]Login failed for user 'marko'. (18456) (SQLDriverConnect); [28000] [Microsoft][ODBC Driver 17 for SQL Server][SQL Server]Login failed for user 'marko'. (18456)")
~
192.168.25.221 - - [08/Feb/2024 13:58:04]  "GET /ProjectMgt/0.0.1/Project/GetProjectByReferenceNumber?ReferenceNumber=UNRA/WRKS/2018-2019/00178 http" 
Traceback (most recent call last):
  File "D:\dev\Apps\Project Management\project-management-api\helpers\utils.py", line 80, in _do_select
    cursor = _conn.cursor()
AttributeError: 'NoneType' object has no attribute 'cursor'
~
127.0.0.1 - - [08/Feb/2024 14:00:26]  "GET /ProjectMgt/0.0.1/Project/GetProjectByReferenceNumber?ReferenceNumber=UNRA%2FWRKS%2F2018-2019%2F00178 http" 
Traceback (most recent call last):
  File "D:\dev\Apps\Project Management\project-management-api\helpers\utils.py", line 81, in _do_select
    cursor.execute(query, values)
pyodbc.ProgrammingError: ('42000', "[42000] [Microsoft][ODBC Driver 17 for SQL Server][SQL Server]Could not find stored procedure 'getProjectByReferenceNumber'. (2812) (SQLExecDirectW)")
~
127.0.0.1 - - [08/Feb/2024 14:01:01]  "GET /ProjectMgt/0.0.1/Project/GetProjectByReferenceNumber?ReferenceNumber=UNRA%2FWRKS%2F2018-2019%2F00178 http" 
Traceback (most recent call last):
  File "D:\dev\Apps\Project Management\project-management-api\helpers\utils.py", line 81, in _do_select
    cursor.execute(query, values)
pyodbc.ProgrammingError: ('42000', "[42000] [Microsoft][ODBC Driver 17 for SQL Server][SQL Server]Could not find stored procedure 'getProjectByReferenceNumber'. (2812) (SQLExecDirectW)")
~
127.0.0.1 - - [08/Feb/2024 14:01:46]  "GET /ProjectMgt/0.0.1/Project/GetProjectByReferenceNumber?ReferenceNumber=UNRA%2FWRKS%2F2018-2019%2F00178 http" 
Traceback (most recent call last):
  File "D:\dev\Apps\Project Management\project-management-api\helpers\utils.py", line 48, in pyodbc_connect
    database=config.db_name
pyodbc.InterfaceError: ('28000', "[28000] [Microsoft][ODBC Driver 17 for SQL Server][SQL Server]Login failed for user 'marko'. (18456) (SQLDriverConnect); [28000] [Microsoft][ODBC Driver 17 for SQL Server][SQL Server]Login failed for user 'marko'. (18456)")
~
127.0.0.1 - - [08/Feb/2024 14:01:46]  "GET /ProjectMgt/0.0.1/Project/GetProjectByReferenceNumber?ReferenceNumber=UNRA%2FWRKS%2F2018-2019%2F00178 http" 
Traceback (most recent call last):
  File "D:\dev\Apps\Project Management\project-management-api\helpers\utils.py", line 80, in _do_select
    cursor = _conn.cursor()
AttributeError: 'NoneType' object has no attribute 'cursor'
~
